README.md

```diff
@@ -1,13 +1,31 @@
-# s3fs
+# s3fs - What it is
 
 s3fs allows Linux, macOS, and FreeBSD to mount an S3 bucket via FUSE.
+FUSE (Filesystem in Userspace), is used instead of a kernel based Filesystem, as they are much easier to debug and work with.
```

Comment:
I think that this additional sentence and the following additional sentence to `Features` should be separate items.
This is because I think it's better to separate the text for users who just want to mount a bucket and are unfamiliar with FUSE, etc., and for users who can understand them.

I will consider the separation including the current `Features`.
For example, in `Features`, explain that user can simply mount an S3 bucket to user's filesystem and work with it.
Then another section(or sub-section) should detail how s3fs handles and supports S3 server objects/etc.

Maybe I think it's better.